src/api/devtools.ts

```diff
@@ -21,9 +26,61 @@ const sendMessageToBackground = (message: String) => {
     })
 }
 const getSourceCode = () => {
+    
+    console.log("Getting the React code");
+
+    chrome.devtools.inspectedWindow.getResources(e => e.filter(obj => {
+        if (obj.url.includes("src") && obj.url.includes("localhost")) {
+            obj.getContent(e => {
+                console.log(e); 
+                appendToStorage("jsCode", e)
+            })
+        }
+    }));
+    
+    console.log("Getting CSS files");
+    chrome.devtools.inspectedWindow.getResources(function(resources:any) {
+        for(var i = 0; i < resources.length; i++)
+        {
+            if (resources[i].type === 'stylesheet') {
+                console.log("found CSS file!");
+                resources[i].getContent(function(content:any) {
+                    console.log(content); 
+                    appendToStorage("cssCode", content)
+                });
+                console.log(resources[i]);
+            }
+        }
+    });
+
+    console.log("Getting HTML file");
+    chrome.devtools.inspectedWindow.getResources(function(resources:any) {
+        for(var i = 0; i < resources.length; i++)
+        {
+            if (resources[i].type === 'document') {
+                console.log("found HTML file!");
+                resources[i].getContent(function(content:any) {
+                    console.log(content); 
+                    appendToStorage("htmlCode", content)
+                });
+                console.log(resources[i]);
+            }
+        }
+    });
+}
+
+const parseJSCode = () => {
+    console.log("Parsing code...");
     chrome.devtools.inspectedWindow.getResources(e => e.filter(obj => {
-        if (!obj.url.includes("node_modules") && obj.url.includes("localhost"))
-            obj.getContent(e => console.log(e))
-    }))
+        if (obj.url.includes("src") && obj.url.includes("localhost")) {
+            obj.getContent(e => {
+                console.log(e); 
```

Comment:
Remove. There should be two functions. one that reads the file and another that parses the file. We already have one that read js file. we should pass the js file to parseJSCode(JScode) and then parse it.